src/server/services/redis.js

```diff
@@ -128,6 +128,16 @@ export async function getUser(username) {
     return client.hgetall(`user:${username}`);
 }
 
+export async function getAllUsers() {
+    const usernames = await getAllUserNames();
+    return Promise.all(usernames.map(async (username) => await client.hgetall(`user:${username}`)));
+}
+
+export async function getAllUserNames() {
+    const keys = await client.keys('*');
```

Comment:
Try to use `client.keys('user:*');` instead, I think that will work

Reply:
I have changed it to be `client.keys('user:*');` :)